src/app/components/filters/filters.test.tsx

```diff
@@ -0,0 +1,96 @@
+import { AdditionalFilters, Stops } from "../../models/flight-parameters";
+import Filters, { FiltersProps } from "./filters";
+import { PriceRangeFilterProps } from "./price-range-filter/price-range-filter";
+import { StopsFilterProps } from "./stops-filter/stops-filter";
+import { fireEvent, render, screen } from "@testing-library/react";
+
+enum MockedComponentTestIds {
+  StopsFilter = 'stops-filter',
+  PriceRangeFilterProps = 'price-range-filter'
+}
+
+jest.mock('./filters.css', () => {});
+
+jest.mock('../../../components/ui/card', () => ({
+  Card: ({ children }: any) => <div>{...children}</div>
+}));
+
+jest.mock('./stops-filter/stops-filter', () => ({
```

Comment:
I would recommend to have the mocks separately rather than keeping it along with the tests

Reply:
Can you help me out with this?